Source/WebCore/loader/FrameLoader.cpp

```diff
@@ -3155,6 +3155,15 @@ void FrameLoader::checkLoadComplete(LoadWillContinueInAnotherProcess loadWillCon
             frames.append(localFrame.releaseNonNull());
     }
 
+    // Provisional frames that are not in the frame tree need to be included to report provisional load failures.
+    if (m_frame->settings().siteIsolationEnabled()) {
+        bool containsThisFrame = std::ranges::any_of(frames, [thisFrame = Ref { m_frame.get() }] (auto& frame) {
```

Comment:
nit - you don't need to ref m_frame

Reply:
I agree, but the safer-cpp bot complained